internal/provider/schema_cloudflare_argo_tunnel.go

```diff
@@ -24,5 +24,9 @@ func resourceCloudflareArgoTunnelSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		"tunnel_token": {
```

Comment:
isn't this the same as `secret` that we already have the schema?

Reply:
no, this is the token used by hosts to authenticate and run tunnels. this does technically include the `secret` but also the account id and tunnel id, all being base64 encoded